thrust/cmake/ThrustMultiConfig.cmake

```diff
@@ -6,14 +6,14 @@ function(thrust_configure_multiconfig)
   option(THRUST_ENABLE_MULTICONFIG "Enable multiconfig options for coverage testing." OFF)
 
   # Dialects:
-  set(THRUST_CPP_DIALECT_OPTIONS
```

Comment:
important: same note, it's a cmake option, should stay as `THRUST_CPP_DIALECT_OPTIONS`